includes/admin/donors/donors.php

```diff
@@ -217,7 +217,10 @@ function give_donor_view( $donor ) {
 
 					<div id="donor-name-wrap" class="left">
 						<span class="donor-id">#<?php echo $donor->id; ?></span>
-						<span class="donor-name info-item edit-item"><input size="15" data-key="name" name="customerinfo[name]" type="text" value="<?php echo esc_attr( $donor->name ); ?>" placeholder="<?php esc_attr_e( 'Donor Name', 'give' ); ?>" /></span>
+						<span class="donor-name info-item edit-item">
+							<input size="15" data-key="first_name" name="customerinfo[first_name]" type="text" value="<?php echo esc_attr( $donor->get_first_name() ); ?>" placeholder="<?php esc_attr_e( 'First Name', 'give' ); ?>" />
```

Comment:
@mehul0810 `esc_attr` used for dynamic value about which we are not sure that is it safe to use in attribute or not. But here `Last Name` and `First Name` is safe string to use as attribute, so no need of `esc_attr_e`, use `_e` instead.